### PR TITLE
Add liveness and readiness probe

### PR DESCRIPTION
**Pull request by lxlxok**
1. Expose the healthz handler.
2. Add liveness and readiness probe in deployment file.

### CODE REVIEW

**Comment by thomashargrove** [comment]
Thanks Xiao!

File: helm/templates/statefulset.yaml [hunk 1]
```diff
@@ -29,6 +29,28 @@ spec:
...
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.livenessProbe.successThreshold }}
+          failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /
```
Comment (mallow111) [inline helm/templates/statefulset.yaml]: Referred the example here: https://github.com/helm/charts/blob/master/stable/vault-operator/templates/deployment.yaml#L55, here the path should be `/healthz`?

**Comment by mallow111** [comment]
@lxlxok  Thanks for adding livenessProbe and readinessProbe, one minor pick, also you may want to wait an email [`GitHub] @amateurhuman has invited you to join the @salesforce organization` flow in your github email, it will fix the CLA issue.

File: helm/templates/statefulset.yaml (hunk 1 above)
Comment (lxlxok) [reply helm/templates/statefulset.yaml]: Hi @mallow111  Thanks for your review. I updated the path from `/` to `/healthz`.

**Pull request merged**